content/en/docs/security/best-practices.md

```diff
@@ -239,7 +250,6 @@ The recommendations below are based on Flux latest version.
     - Check whether there are Admission Controllers/OPA rules blocking changes in Platform Admin's permissions and overall resources.
   </details>
 
-
-[Kubernetes CIS Benchmark]: https://www.cisecurity.org/benchmark/kubernetes
-[RuntimeClass]: https://kubernetes.io/docs/concepts/containers/runtime-class/
-[Taints and Tolerations]: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+[kubernetes cis benchmark]: https://www.cisecurity.org/benchmark/kubernetes
+[runtimeclass]: https://kubernetes.io/docs/concepts/containers/runtime-class/
+[taints and tolerations]: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
```

Comment:
We can remove the links from here if we are already adding them directly where they are being used.